backend/src/graphql/resolver/AdminResolver.ts

```diff
@@ -696,4 +700,46 @@ export class AdminResolver {
     logger.debug(`updateContributionLink successful!`)
     return new ContributionLink(dbContributionLink)
   }
+
+  @Authorized([RIGHTS.ADMIN_CREATE_CONTRIBUTION_MESSAGE])
+  @Mutation(() => ContributionMessage)
+  async adminCreateContributionMessage(
+    @Args() { contributionId, message }: ContributionMessageArgs,
+    @Ctx() context: Context,
+  ): Promise<ContributionMessage> {
+    const user = getUser(context)
+    const queryRunner = getConnection().createQueryRunner()
+    await queryRunner.connect()
+    await queryRunner.startTransaction('READ UNCOMMITTED')
+    const contributionMessage = DbContributionMessage.create()
+    try {
+      const contribution = await Contribution.findOne({ id: contributionId })
+      if (!contribution) {
+        throw new Error('Contribution not found')
+      }
```

Comment:
```suggestion
      if (!contribution) {
        logger.error(`Contribution not found!`)
        throw new Error('Contribution not found!')
      }
```

Do we want logging aswell here?

Reply:
But it's in a try so it will be catched in the following catch where it is already logged. Or?

But we would get a loggin with more details …

Reply:
But this error message seems to be logged in the following catch as well … 🤔